src/components/Tooltip/Tooltip.tsx

```diff
@@ -253,7 +253,7 @@ const Tooltip = ({
     mouseHandler,
     "mousemove",
     true,
-    followMouse && isOpen,
+    followMouse,
```

Comment:
```suggestion
  useListener(wrapperRef.current, mouseHandler, "mousemove", true, followMouse);
```
This whole block should be in a single line now.